documentation/airport-format.md

```diff
@@ -306,26 +311,27 @@ They're used when we need aircraft to fly over a location that doesn't have an a
 ```javascript
 "_AUTUM220015": [42.324333, -71.736833]
 ```
-* Any fixes desired a given distance out on final of a given runway will be described via the distance from the threshold. This would be the runway name, two digit distance in nautical miles, then `DME`. All of these should be marked as RNAV fixes (via the underscore prefix).
+* Any fixes desired a given distance out from a given runway will be described via the distance from the threshold. This would be the runway name for which the procedure is available, two digit distance in nautical miles, then `DME`. All of these should be marked as RNAV fixes (via the underscore prefix).
 ```javascript
 "_RWY33L01DME": [42.342838, -70.975751]
 ```
 
 ### Restricted Airspace
```

Comment:
Before leaving the fix section, let's add a section about something you've frequently had to do: intersections of radials off two fixes, and intersections between runway departure courses and fix radials, and intersections between a radial and a DME arc.

---

Intersections between radials from two fixes:
Fixes may be defined based on the intersection between _outbound_ radials from two defined fixes. For a point northeast of FIXXA, and northwest of FIXXB, we could create `_FIXXA050FIXXB320`, where the three digit numbers after the fix names are the direction from that fix to the described location.

Intersections between runway heading and a fix radial:
Fixes may be defined based on the intersection of a runway's outbound course and an _outbound_ radial of any fix. For a point aligned with Tampa's 19R departure path and the PIE VOR's outbound radial 116, we get `"_RWY19RPIE116": [27.848198, 82.546200]`. Note that if the intersection were to be on the arrival half of a given runway, the opposite runway should be used to keep with the convention of using the departure course: `"_RWY1LPIE042": [28.077769, -82.538636]`.

Intersections between a fix radial and a given distance from _another_ fix:
Fixes may be defined based on the intersection of a fix's outbound radial and the DME arc of the specified distance from a separate fix. This is formatted like `_FIXXA050FIXXB05DME`, where the first fix has a three digit outbound radial, and the second fix has a two-digit distance in nm, followed by `DME`. Similarly, this can be done with runways using the same patterns as before `_RWY22LFIXXB05DME`.

Reply:
done